net/redisclient.go

```diff
@@ -87,6 +101,96 @@ const (
 	defaultConnMetricsInterval = 60 * time.Second
 )
 
+// JumpHash
+// https://arxiv.org/pdf/1406.2294.pdf
+type JumpHash struct {
+	hash   hash.Hash64
+	shards []string
+}
+
+func NewJumpHash(shards []string) redis.ConsistentHash {
+	return &JumpHash{
+		hash:   fnv.New64(),
+		shards: shards,
+	}
+}
+
+func (j *JumpHash) Get(k string) string {
+	_, err := j.hash.Write([]byte(k))
```

Comment:
Is it safe to use the same hash instance from multiple threads?
I think we have to create a new hash instance on the fly here.

Reply:
true, will fix it

Reply:
done

Reply:
I think we need to create hash instance for each new key, i.e. inside `Get`

Reply:
yeah thanks, I had the fix, but reverted the file before pushing :D

Reply:
pushed the fix